README.md

```diff
@@ -107,12 +101,64 @@ Finally, properties are what give the selected elements of a rule declaration th
 
 ### Comments
 
-* Prefer line comments (`//` in Sass-land) to block comments.
+* Prefer line comments (`//`) to block comments.
 * Prefer comments on their own line. Avoid end-of-line comments.
 * Write detailed comments for code that isn't self-documenting:
   - Uses of z-index
   - Compatibility or browser-specific hacks
 
+### Colors
+
+When implementing feature styles, you should only be using color variables provided by theme.css
+
+When adding a color variable to theme.css, only use colors available to you by
+the color palette. Those should be listed to you in a separate file called
+`color_palette.css`.
+
+Using HEX color units is preferred over RGB, RGBA, named, HSL, or HSLA values.
+
+**Bad**
```

Comment:
Let's move this block down so the examples are always in the Good then Bad order.